.github/workflows/vulncheck.yml

```diff
@@ -27,9 +27,9 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v4
         with:
+          cache: false # setup-go v4 caches by default
```

Comment:
Un-neccesary comment/change

Reply:
Added so we do not use a cache for any of the `setup-go@v4` actions. See the relevant commit and https://github.com/actions/setup-go/blob/main/action.yml#L15-L17